### PR TITLE
MacOS 12.x Installation notes added to readme

### PR DESCRIPTION
**Pull request by damoore044**

### CODE REVIEW

**Review by JacobCallahan — APPROVED** [review]
This looks good from a content perspective. However, I'd like to have @jyejare verify the installation instructions.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,38 @@ Broker can also be ran outside of its base directory. In order to do so, specify
 `BROKER_DIRECTORY` envronment variable.
 ```BROKER_DIRECTORY=/home/jake/Programming/broker/ broker inventory```
 
+# Installation MacOS 12.x
+```
+brew install cmake
+brew install openssl
+brew install libssh2
+cd <broker root directory>
+pip install . or pip install broker
```
Comment (Griffin-Sullivan) [inline README.md:38]: Do we want to make the distinction here that `pip install .` will install locally where as `pip install broker` will install latest from PyPI?

Comment (JacobCallahan) [reply README.md:38]: Good idea, for those unfamiliar with the difference

Comment (Griffin-Sullivan) [reply README.md:38]: yeah I think that would be a nice change if you don't mind adding it

**Review by Griffin-Sullivan — APPROVED** [review]
Looks good!

File: README.md [hunk 2]
```diff
@@ -12,11 +12,51 @@ Broker is a tool designed to provide a common interface between one or many serv
...
+brew install cmake
+brew install openssl
+brew install libssh2
+cd <broker root directory>
+
+pip install .   ( install Broker downloaded locally )   
+or   
+pip install broker   ( install latest version from PyPI )
+
+pip install ssh2-python
```
Comment (ogajduse) [inline README.md:47]: If you install broker you should not need to install `ssh2-python` additionally as this lib is a broker's requirement.

**Review by ogajduse — APPROVED** [review]
ACK, one comment left.

**Comment by ogajduse** [comment]
@JacobCallahan Why are my checkmarks in my broker reviews grey while others have green? :thinking: :thinking: Hmmm

**Comment by JacobCallahan** [comment]
> @JacobCallahan Why are my checkmarks in my broker reviews grey while others have green? :thinking: :thinking: Hmmm

haha it means you should be off and not paying attention right now

File: README.md [hunk 3]
```diff
@@ -12,7 +12,11 @@ Broker is a tool designed to provide a common interface between one or many serv
 ```
 dnf install cmake
 cd <broker root directory>
-pip install .   or   pip install broker
+
+pip install .   ( install Broker downloaded locally )
```
Comment (jyejare) [inline README.md:16]:
```suggestion
pip install .   ( install Broker cloned locally )   
```

File: README.md [hunk 4]
```diff
@@ -29,6 +33,42 @@ Broker can also be ran outside of its base directory. In order to do so, specify
 `BROKER_DIRECTORY` envronment variable.
 ```BROKER_DIRECTORY=/home/jake/Programming/broker/ broker inventory```
 
+# Installation MacOS 12.x
```
Comment (jyejare) [inline README.md:36]:
```suggestion
# Installation MacOS 12.x +
```

File: README.md [hunk 5]
```diff
@@ -29,6 +33,42 @@ Broker can also be ran outside of its base directory. In order to do so, specify
...
+```
+brew install cmake
+brew install openssl
+brew install libssh2
+cd <broker root directory>
+
+pip install .   ( install Broker downloaded locally )
```
Comment (jyejare) [inline README.md:43]:
```suggestion
pip install .   ( install Broker cloned locally )   
```

File: README.md [hunk 6]
```diff
@@ -29,6 +33,42 @@ Broker can also be ran outside of its base directory. In order to do so, specify
...
+Then edit the broker_settings.yaml file
+
+Error: If libsshX.Y.dylib cannot be found, follow the additional steps:
+
+- Locate the .dylib files from within libssh2 (likely in `/opt/homebrew/Cellar/` or `/usr/lib/`)
+- Export the path to the .dylibs directory as the 
+`DYLD_LIBRARY_PATH` environment variable;
+
+```export DYLD_LIBRARY_PATH=/opt/homebrew/Cellar/libssh2/1.10.0/lib/```
+- Place the export statement in your working shell config; .bashrc .zshrc etc
```
Comment (jyejare) [inline README.md:58]:
```suggestion
- Place the export statement in your working shell config; .zshrc etc
```
Mac does provides zshrc only I think !

**Review by jyejare — CHANGES_REQUESTED** [review]
Some nitpick comments! 

@damoore044 I would also include the workaround for pycurl error and that is nothing but to uninstall existing pycurl and reinstall with command:
```
# PYCURL_SSL_LIBRARY=openssl LDFLAGS="-L$(brew --prefix openssl)/lib" CPPFLAGS="-I$(brew --prefix openssl)/include" pip install --compile --install-option="--with-openssl" pycurl
```
That uses openssl library for pycurl installation than the default one.

File: README.md (hunk 6 above)
Comment (damoore044) [reply README.md:58]: Mac ships with zsh, but bash can be brew installed and set as the main shell. Should we just have zsh since that is the default?

**Review by jyejare — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (jyejare) [reply README.md]: No Worries, we can mention both of them !

**Pull request merged**